docs/source/remote-data-services.rst

```diff
@@ -1,3 +1,74 @@
+Data Ingestion
+``````````````
+
+Internal Data Ingestion
+=======================
```

Comment:
I think that this section is more developer focused, and would be better at the bottom of this page, after all of the user-focused content.